src/cli.ts

```diff
@@ -3,7 +3,7 @@ import type { Argv } from 'yargs'
 import yargs from 'yargs'
 import { hideBin } from 'yargs/helpers'
 import c from 'picocolors'
-import { version } from '../package.json'
```

Comment:
Since we've disabled the `@rollup/plugin-json`'s `namedExports` optimization, we can't use named import here.